app/controllers/sessions_controller.rb

```diff
@@ -1,14 +1,20 @@
 class SessionsController < ApplicationController
   skip_before_action :require_login
-  
+
   def create
-    if user = User.from_google_auth(request.env["omniauth.auth"])
+    user = User.from_google_auth(request.env["omniauth.auth"])
+    if user && user.role == "default"
       session[:user_id] = user.id
       flash[:success] = "Welcome, gardener, we're glad to have you among the roses."
       redirect_to dashboard_path
+    else
+      user && user.role == "admin"
```

Comment:
should this be `elsif user && user.role == "admin"`? Else seems like line 11 doesn't really do anything - just evaluates to true/false, but doesn't affect any other code.

Reply:
good catch @asmolentzov

Reply:
Totally agree! I'm going to make the changes now